content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -542,12 +542,26 @@ jobs:
         uses: docker://alpine:3.8
 ```
 
-##### Example using a Docker public registry action
+##### Example using a Github Container Registry
```

Comment:
There's some reusable text available for this product name:
```suggestion
##### Example using the {% data variables.product.prodname_github_container_registry %}
```

Reply:
Cool :-) I was not aware of it.